cmd/process-agent/main_common.go

```diff
@@ -74,10 +74,10 @@ func main() {
 func runAgent(globalParams *command.GlobalParams, exit chan struct{}) {
 	cleanupAndExit := cleanupAndExitHandler(globalParams)
 
-	if !globalParams.Info && globalParams.PidFilePath != "" {
+	if globalParams.PidFilePath != "" {
 		err := pidfile.WritePID(globalParams.PidFilePath)
 		if err != nil {
-			log.Errorf("Error while writing PID file, exiting: %v", err)
+			_ = log.Errorf("Error while writing PID file, exiting: %v", err)
```

Comment:
can we remove this no-op assignment if we're not doing anything with the return value, similar to what was happening before?

Reply:
TBH this is a style point that has been a bit contentious for me. My IDE usually tells me to explicitly ignore errors, and I've seen it in a couple style guides as well. I'm happy to remove it though